languages/python/oso/polar/partial.py

```diff
@@ -18,15 +23,9 @@ def dot_path(expr):
     _this.created_by => ('created_by',)
     _this.created_by.username => ('created_by', 'username')"""
 
-    if isinstance(expr, Variable) and expr != Variable("_this"):
+    if isinstance(expr, Variable):
```

Comment:
Are SQLAlchemy tests still passing with these changes? Is the doc comment above accurate still?

Reply:
I doubt it, and no

Reply:
updated